pkgs/tools/filesystems/squashfs/default.nix

```diff
@@ -22,6 +22,7 @@ stdenv.mkDerivation rec {
     ./0001-If-SOURCE_DATE_EPOCH-is-set-override-timestamps-with.patch
     ./0002-If-SOURCE_DATE_EPOCH-is-set-also-clamp-content-times.patch
     ./0003-remove-frag-deflator-thread.patch
```

Comment:
Can you move the comment from above directly to this patches and make a newline after them so it is clear that the 4k align patch is not meant for reproducibility?